pyfstat/mcmc_based_searches.py

```diff
@@ -1233,37 +1233,8 @@ def plot_cumulative_max(self, **kwargs):
 
         if hasattr(self, "search") is False:
             self._initiate_search_object()
-        if self.binary is False:
-            self.search.plot_twoF_cumulative(
-                self.label,
-                self.outdir,
-                F0=d["F0"],
-                F1=d["F1"],
-                F2=d["F2"],
-                Alpha=d["Alpha"],
-                Delta=d["Delta"],
-                tstart=self.minStartTime,
-                tend=self.maxStartTime,
-                **kwargs,
-            )
-        else:
-            self.search.plot_twoF_cumulative(
-                self.label,
-                self.outdir,
-                F0=d["F0"],
-                F1=d["F1"],
-                F2=d["F2"],
-                Alpha=d["Alpha"],
-                Delta=d["Delta"],
-                asini=d["asini"],
-                period=d["period"],
-                ecc=d["ecc"],
-                argp=d["argp"],
-                tp=d["argp"],
-                tstart=self.minStartTime,
-                tend=self.maxStartTime,
-                **kwargs,
-            )
+
+        self.search.plot_twoF_cumulative(self.label, self.outdir, **d, **kwargs)
```

Comment:
I don't understand how this call works with the rewritten core function - shouldn't it expect a non-optional keyword argument `CFS_input` but instead you're passing it the **ed version of the dictionary, i.e. the tuple of signal parameters...? Why does that work? :confused: 

Also, are we now never allowing PFS from this call? No, wait, I guess the end user can pass `PFS_input` through the `kwargs` of this one, right?

Reply:
...hey, github, what's going on? This was shown to me as the up-to-date diff after I refreshed, Rodrigo has not pushed any new changes since 5h ago, but now it's "outdated". :confused: :confounded: 

In fact, the new version looks much more like what I expected!